sanic/sessions.py

```diff
@@ -0,0 +1,303 @@
+# -*- encoding: utf-8 -*-
+
+import hashlib
+
+from itsdangerous import URLSafeTimedSerializer
+from datetime import datetime
+
+import ujson as json
+from sanic.exceptions import BadSignature
+
+
+def total_seconds(td):
+    """Returns the total seconds from a timedelta object.
+    :param timedelta td: the timedelta to be converted in seconds
+    :returns: number of seconds
+    :rtype: int
+    """
+    return td.days * 60 * 60 * 24 + td.seconds
+
+
+class SessionMixin(object):
```

Comment:
Sorry, I am not suggesting to build a whole thing.  If there has been already one, let's use it. BUT, Make it sharable, like a lib, instead of copying the code. It's not good for maintaining the code base in both sides. Someone should always track changes to check if any bug or upgrade, it's not good pattern, i'd like to see sth has global impact. That's my whole point.